wordCount.rb

```diff
@@ -0,0 +1,32 @@
+class WordCounter
+  attr_accessor :file, :output
+  def initialize(file)
+    @file = file
+    @output = {}
+    self.read
```

Comment:
Think about the API you're creating by chaining together these methods. Some things to think about include, 1) Do you want WordCounter.new(file) to print out the word count? 2) How would you test the individual methods here?

I would recommend writing some rspec tests to get a feel for your API. When writing m tests, I try to write the code that I want to have. I then fill in the blanks inside of the class/module.